virtual_rainforest/models/animals/dummy_animal_module.py

```diff
@@ -63,3 +69,41 @@ class PalatableSoil:
     """The amount of energy in the soil pool [J]."""
     position: int
     """The grid position of the soil pool."""
+
+
+class AnimalCohort:
+    """This is a class of animal cohorts."""
+
+    def __init__(self, name: str, mass: float, age: float, position: int) -> None:
+        """The constructor for the Animal class."""
+        self.name = name
+        """The functional type name of the animal cohort."""
+        self.mass = mass
+        """The average mass of an individual in the animal cohort [kg]."""
+        self.age = age
+        """The age of the animal cohort [days]."""
+        self.position = position
+        """The grid position of the animal cohort."""
+        self.individuals: int = ceil(
+            DamuthsLaw.coefficienct * self.mass ** (DamuthsLaw.exponent)
+        )
+        """The number of individuals in the cohort."""
+        self.is_alive: bool = True
```

Comment:
Purely out of curiosity: moving forward were you thinking of defining "death" as a proportion of dead individuals in a cohort? Or does it make sense for the modelling to have it as a binary attribute?

Reply:
There are two versions of 'death' that I'll be dealing with: the death of individuals within a cohort and the final death of a cohort as it reaches zero individuals. The is_alive bool may not end up being necessary long term but I think is useful in the interim to make sure that 'dead' cohorts are behaving and are tracked properly. Does that make sense?

Reply:
Yep! I was just interested to know what your plans were.